apps/bynder/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "1.1.16",
   "private": true,
   "devDependencies": {
-    "@contentful/app-scripts": "1.32.0",
+    "@contentful/app-scripts": "^2.1.0",
     "@vitejs/plugin-react": "^4.3.4",
     "happy-dom": "^15.11.7",
-    "vite": "^5.4.11",
-    "vitest": "^2.1.8"
+    "vite": "^6.2.0",
+    "vitest": "^3.0.7"
   },
   "dependencies": {
     "@contentful/dam-app-base": "^3.0.9",
-    "react": "18.3.1",
-    "react-dom": "18.3.1"
+    "react": "18.2.0",
```

Comment:
when I tried to `npm i` from the bynder root, I was getting a dependency conflict due to the react version. seems like dependabot updated it _6 months_ ago. I don't quite understand how it was passing.  I looked at our `dam-app-base` package and it's using React 18.2 so I put this back to resolve.